Example/AppDelegate+CarPlay.swift

```diff
@@ -104,7 +104,7 @@ extension AppDelegate: CPListTemplateDelegate {
         // Selected a favorite
         if let userInfo = item.userInfo as? [String: Any],
             let waypoint = userInfo[CarPlayManager.CarPlayWaypointKey] as? Waypoint {
-            carPlayManager.calculateRouteAndStart(to: waypoint, completionHandler: completionHandler)
+            carPlayManager.preview(waypoints: [waypoint], completionHandler: completionHandler)
```

Comment:
This method was renamed to `previewRoutes(between:completionHandler:)`. Fixed in cc255c4d8b66bfab1150e321e696d54cf4debe76.